pyomo/contrib/doe/doe.py

```diff
@@ -107,56 +131,34 @@ def __init__(
         else:
             self.solver = self._get_default_ipopt_solver()
 
-        # time set name
-        self.t = time_set_name
-
         # check if discretization is needed
         self.discretize_model = discretize_model
 
         # check if there is prior info
-        self.prior_FIM = prior_FIM
+        if prior_FIM is None:
+            self.prior_FIM = np.zeros((len(self.param), len(self.param)))
+        else:
+            self.prior_FIM = prior_FIM
+        self._check_inputs()
 
         # if print statements
         self.logger = logging.getLogger(__name__)
-        self.logger.setLevel(level=logging.WARN)
+        self.logger.setLevel(level=logging.INFO)
```

Comment:
Check all files and doc strings for a verbose flag. Instead switch to use logging. For example, look at the `_print` function for results.